tests/test_infer/test_chatglm2_infer.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import numpy as np
+import pytest
+import torch
+import torch.distributed as dist
+from packaging import version
+from transformers import AutoModel, AutoTokenizer
+
+import colossalai
+from colossalai.cluster import ProcessGroupMesh
+from colossalai.inference.tensor_parallel.engine import TPInferEngine
+from colossalai.logging import disable_existing_loggers
+from colossalai.shardformer import ShardConfig, ShardFormer
+from colossalai.shardformer.modeling.chatglm2_6b.modeling_chatglm import ChatGLMForConditionalGeneration, ChatGLMModel
+from colossalai.testing import clear_cache_before_run, parameterize, rerun_if_address_is_in_use, spawn
+
+os.environ['TRANSFORMERS_NO_ADVISORY_WARNINGS'] = 'true'
+TPSIZE = 1
+BATCH_SIZE = 8
+MAX_INPUT_LEN = 12
+MAX_OUTPUT_LEN = 100
+CUDA_SUPPORT = version.parse(torch.version.cuda) > version.parse('11.5')
+
+
+@parameterize('test_config', [{
+    'tp_size': TPSIZE,
+}])
+def run_chatglm2_test(test_config):
+
+    chatglm2_model_path = "/home/lccjh/data2/lccjh/chatglm2-6b"
```

Comment:
Prevent using absolute path on server. (An alternative is to use model zoo)

Reply:
done!